helper/general_objects/parsers.go

```diff
@@ -391,6 +391,9 @@ func (constraint *ArrayConstraint[T]) ToMap() map[string]any {
 		constraintMap["min_length"] = constraint.MinLength
 		constraintMap["max_length"] = constraint.MaxLength
 		constraintMap["pattern"] = constraint.Pattern
+	case "BINARY":
+		constraintMap["min_length"] = constraint.MinLength
+		constraintMap["max_length"] = constraint.MaxLength
```

Comment:
@leanspace/blue-team This fixes a change detection in activity definitions for binary array